src/lib/dat-cp.js

```diff
@@ -135,6 +135,15 @@ export default class DatCp {
     })
   }
 
+  async setDownloadDest(path) {
```

Comment:
I think instead of making needing to make a separate call to this, there should just be a possible `dest` property in the `options` passed to the constructor which causes this method to be called.